lib/elements/reinforcement/straight_longitudinal.rb

```diff
@@ -0,0 +1,80 @@
+require 'elements/reinforcement/base'
+require 'elements/reinforcement/straight_longitudinal_layer'
+require 'errors/reinforcement/empty_layers'
+
+module Elements
+  module Reinforcement
+    class StraightLongitudinal < Base
+      attr_accessor :layers
+
+      def initialize(z_base:, distribution_direction:, direction: 1)
+        @z_base = z_base
+        @direction = direction
```

Comment:
Let's rename these for clarity
```suggestion
      def initialize(reference_plane_height:, distribution_direction:, above_reference_plane: true)
        @reference_plane_height = reference_plane_height
        @above_reference_plane = above_reference_plane
```

Reply:
fuck you

Reply:
## new